src/diffpy/cmi/cli.py

```diff
@@ -72,6 +72,43 @@ def map_pack_to_examples() -> dict[str, List[str]]:
     return examples_by_pack
 
 
+def copy_examples(
+    examples: List[str],
+    target_dir: Optional[Path] = None,
+) -> List[Path]:
+    """Copy one or more examples to a target directory.
```

Comment:
how about:
`"""Copy one or more examples from the installed package to a target directory.`